frontend/src/context/loading/LoadingState.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+import loadingContext from './loadingContext';
+
+const LoadingState = (props) => {
```

Comment:
Suggest to change the file extension to tsx as well and add types.
```suggestion
const LoadingState = (props: React.PropsWithChildren) => {
```